doc/configtables/electricity.csv

```diff
@@ -1,7 +1,7 @@
 ,Unit,Values,Description
-voltages,kV,"Any subset of {200., 220., 300., 380., 500., 750.}",Voltage levels to consider
+voltages,kV,"Any subset of {220., 300., 330., 380., 400., 500., 750.}",Voltage levels to consider
 base_network, --, "Any value in {'entsoegridkit', 'osm-prebuilt', 'osm-raw}", "Specify the underlying base network, i.e. GridKit (based on ENTSO-E web map extract, OpenStreetMap (OSM) prebuilt or raw (built from raw OSM data), takes longer."
-osm-prebuilt-version, --, "float, any value in range 0.1-0.4", "Choose the version of the prebuilt OSM network. Defaults to latest Zenodo release."
+osm-prebuilt-version, --, "float, any value in range 0.1-0.5", "Choose the version of the prebuilt OSM network. Defaults to latest Zenodo release."

```

Comment:
It looks like you're changing the zenodo link in this PR to point specifically to 0.5. Since it says here, that it defaults to the "latest release" we could also use a static link that always point to the latest version, right? Not sure if that is possible, but I thought zenodo offered these links that automatically point to the latest version of a dataset.

Reply:
Zenodo does indeed provide a doi link that then redirects to the latest version. 
The reason I implemented it like this is for the user to have more control, i.e. if a previous study was done on an older version, I can keep using X.y.